docs/NcObject.md

```diff
@@ -1,148 +1,52 @@
 # NcObject
 
-NcObject is the base abstract class from which all other NCA classes must inherit from.
+NcObject is the base abstract control class for any control class in the control model. Any other control class MUST be derived directly or indirectly from this class.
 
-The properties of NcObject are listed in the following table.
+The control class model for NcObject is listed in the [Framework](Framework.md#ncobject).
 
-| **Property Name** | **Datatype**                   | **Readonly** | **Description**                                                       |
-| ----------------- | ------------------------------ | ------------ | ----------------------------------------------------------------------|
-| classId           | NcClassID                      | Yes          | sequence of numeric class identifiers or authority keys               |
-| oid               | NcOid                          | Yes          | unique object id                                                      |
-| constantOid       | NcBoolean                      | Yes          | flag to indicate if the oid is constant or not                        |
-| owner             | NcOid                          | Yes          | unique object id of the parent                                        |
-| role              | NcRole                         | Yes          | unique role of the object within the containing block                 |
-| userLabel         | NcString?                      | No           | user definable label                                                  |
-| touchpoints       | sequence<NcTouchpoint>         | Yes          | sequence of touchpoint (see Touchpoints section for details)          |
-
-The `role` is a structural identifier which MUST be persisted across restarts. The role path of an element can be constructed from its role and the roles of all its parents. The role path can then be used to target a particular section of the control model tree or to retrieve the object ids again after a restart.
+The `role` is a structural identifier which MUST be persisted across restarts. asdasda
```

Comment:
Spurious "asdasda" at the end of the line

Reply:
Pretty sure that's a magic word like xyzzy 🧙‍♂️